emretl-runner/assets/start.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+main() {
+  printenv
+  # and startup
+  startup
+}
+
+startup() {
+  envsubst \
+  < /assets/resolver.json.template \
+  > /emr/resolver.json
+  # no files all redirects and aweseme sauce
+  envsubst \
+  < /assets/config.yml.template \
+  | /emr/snowplow-emr-etl-runner --config - \
+  --debug \
+  --config - \
+  --resolver /emr/resolver.json \
+  $EMR_ARGS
```

Comment:
I've stubbed out each of the enrichments but they need the templates to be variabled out and correspond variables set in the env_file or passed in otherwise (as in the case of the rancher templates)